### PR TITLE
Tickets/sitcom 510

### PR DESCRIPTION
**Pull request by isotuela**
Hi Patrick! I look forward to hearing your comments. 

Best regards, 
Ioana

### CODE REVIEW

File: notebooks/summit/AuxTel/SITCOM-510/SITCOM-510_Linearity_test_using_sky_background.ipynb [hunk 1]
```diff
@@ -0,0 +1,715 @@
+{
```
Comment (patrickingraham) [inline notebooks/summit/AuxTel/SITCOM-510/SITCOM-510_Linearity_test_using_sky_background.ipynb:1]: <p>Does this better belong at the top? Essentially cell #2 ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_isotuela/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (patrickingraham) [inline notebooks/summit/AuxTel/SITCOM-510/SITCOM-510_Linearity_test_using_sky_background.ipynb:1]: <p>Might be worth adding a cell before this that says which filters are available.</p><pre class="ql-syntax" spellcheck="false">inst_setup = await script.latiss.get_available_instrument_setup()
logger.info(f'filters are: {inst_setup[0]},\ngratings are: {inst_setup[1]}')
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_isotuela/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Comment (patrickingraham) [inline notebooks/summit/AuxTel/SITCOM-510/SITCOM-510_Linearity_test_using_sky_background.ipynb:1]: <p>you could add cwfs_radius=X if you want to specify it, then pass the <code>radius</code>  argument in the next cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_isotuela/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Comment (patrickingraham) [inline notebooks/summit/AuxTel/SITCOM-510/SITCOM-510_Linearity_test_using_sky_background.ipynb:1]: <p>I think you can remove this cell and the next one?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_isotuela/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Comment (patrickingraham) [inline notebooks/summit/AuxTel/SITCOM-510/SITCOM-510_Linearity_test_using_sky_background.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    await script.atcs.slew_icrs(ra = targets[month]['target_ra'], dec = targets[month]['target_dec'], rot_type = RotType.Physical, rot=0)
</pre><p>Why can't you use RotType.PhysicalSky and then not declare a rotator angle? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_isotuela/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Comment (patrickingraham) [inline notebooks/summit/AuxTel/SITCOM-510/SITCOM-510_Linearity_test_using_sky_background.ipynb:1]: <p>comment this as you shouldn't need it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_isotuela/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Comment (patrickingraham) [inline notebooks/summit/AuxTel/SITCOM-510/SITCOM-510_Linearity_test_using_sky_background.ipynb:1]: <p>You might want to add a note that the bias levels are ~15000 cts (or whatever you see in biases, correct that value if i'm wrong!) and that the ccs performs a row-overscan bias subtraction, which means that the value indicated by the CCS is the true counts.</p><p> </p><p>You can word it better than that of course :D</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_isotuela/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Comment (patrickingraham) [inline notebooks/summit/AuxTel/SITCOM-510/SITCOM-510_Linearity_test_using_sky_background.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #7.        logger.info(f'5sec darks in loop {i} are {darks_5sec}')
</pre><p>I think the <code>darks_5sec</code> is going to be an object, did you want to just report the obs_ids?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_isotuela/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>

**Comment by patrickingraham** [comment]
This is really good! I made some nit-picky comments but it's in great shape.

File: notebooks/summit/AuxTel/SITCOM-510/SITCOM-510_Linearity_test_using_sky_background.ipynb (hunk 1 above)
Comment (isotuela) [reply notebooks/summit/AuxTel/SITCOM-510/SITCOM-510_Linearity_test_using_sky_background.ipynb:1]: <p>Yes, it does!</p>

Comment (isotuela) [reply notebooks/summit/AuxTel/SITCOM-510/SITCOM-510_Linearity_test_using_sky_background.ipynb:1]: <p>Good idea. </p>

Comment (isotuela) [reply notebooks/summit/AuxTel/SITCOM-510/SITCOM-510_Linearity_test_using_sky_background.ipynb:1]: <p>Will do. </p>

Comment (isotuela) [reply notebooks/summit/AuxTel/SITCOM-510/SITCOM-510_Linearity_test_using_sky_background.ipynb:1]: <p>:+1</p>

Comment (isotuela) [reply notebooks/summit/AuxTel/SITCOM-510/SITCOM-510_Linearity_test_using_sky_background.ipynb:1]: <p>Yes!</p>

Comment (isotuela) [reply notebooks/summit/AuxTel/SITCOM-510/SITCOM-510_Linearity_test_using_sky_background.ipynb:1]: <p>Much better</p>

Comment (isotuela) [reply notebooks/summit/AuxTel/SITCOM-510/SITCOM-510_Linearity_test_using_sky_background.ipynb:1]: <p>:+1</p><p> </p>

Comment (isotuela) [reply notebooks/summit/AuxTel/SITCOM-510/SITCOM-510_Linearity_test_using_sky_background.ipynb:1]: <p>It reports the obsess for logging purposes. </p>

**Pull request merged**